core/services/ocr2/plugins/ocr2keeper/evm21/registry.go

```diff
@@ -34,7 +34,7 @@ import (
 
 const (
 	// defaultAllowListExpiration decides how long an upkeep's allow list info will be valid for.
-	defaultAllowListExpiration = 20 * time.Minute
+	defaultAllowListExpiration = 5 * time.Minute
```

Comment:
can we separate this out as its unrelated to PR

Reply:
done